### ISSUE TITLE
http://scratch.mit.edu/projects/74096796/

### ISSUE THREAD

**Issue by robotu** [nathan/phosphorus#466]
Doesn't work? What's the problem?

http://scratch.mit.edu/projects/74096796/
http://phosphorus.github.io/#74096796
Mozilla/5.0 (Windows NT 10.0; WOW64; rv:40.0) Gecko/20100101 Firefox/40.0

**Comment by htmlgames** [comment]
got it to work after issue with Costume.prototype.render: check just for this.baseLayer (where accessed) only within this fn

**Comment by robotu** [comment]
ok, thanks for reply and good work :-)

**Comment by htmlgames** [comment]
I found another issue with this project regarding collision detection for the sprite named stairs.
It returns a size of 0 for height in the Sprite.prototype.touching so never gets checked correctly
I fixed it by adding two lines  (

        collisionCanvas.width = right - left;
        collisionCanvas.height = top - bottom;
	// collision canvas size should not be zero
	if (collisionCanvas.width == 0) {collisionCanvas.width = 1;}
	if (collisionCanvas.height == 0) {collisionCanvas.height = 1;}

**Comment by tjvr** [comment]
> phosphorus.js:518 Uncaught TypeError: Cannot read property 'style' of undefined

> phosphorus.js:1790 Uncaught TypeError: Cannot read property 'width' of undefined

**Comment by htmlgames** [comment]
Did you try here?
http://homepage.ntlworld.com/infinnerty/differences/UAT/itch/#74096796

**Comment by trumank** [comment]
@htmlgames That only silently fails when it finds an SVG it can't handle. It's not really a fix.

**Comment by htmlgames** [comment]
Fair point - having seen how tricky SVG is in phosphorus (I've been playing around with canvg.js); it at least gets something playable on the screen as a compromise.
